.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,21 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Simulate Docker Build"
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 2 * * *' # run daily
```

Comment:
kinda required, since ubuntu base dependencies keep shifting.

I'd be happier if I did this properly with nixos, but I need to put in the effort to actually get there myself :-)